src/com/lftechnology/java/training/sanish/fibonacci/FibonacciNumber.java

```diff
@@ -0,0 +1,40 @@
+package com.lftechnology.java.training.sanish.fibonacci;
+
+/**
+ * Provide functions to operate with fibonacci number
+ * @author Sanish Maharjan <sanishmaharjan@lftechnology.com>
+ */
+public class FibonacciNumber {
+	/**
+	 * Return sum 0f event fibonacci number with in given max limit
+	 * 
+	 * @param firstNum {@link Integer} first number of fibonacci number series 
+	 * @param secondNum {@link Integer} second number of fibonacci number series 
+	 * @param maxFibonacciNum {@link Integer} maximum fibonacci number
+	 * @return {@link Integer}
+	 * @author Sanish Maharjan <sanishmaharjan@lftechnology.com>
+	 */
+	public static int sumEventFibonacciNumber(int firstNum, int secondNum, int maxFibonacciNum){
+		int nextNum = firstNum + secondNum;
+		int eventFibonacciNumSum = 0;
+		if(firstNum % 2 == 0){
+			eventFibonacciNumSum +=firstNum;
+		}
+		
+		if(secondNum % 2 == 0){
+			eventFibonacciNumSum +=secondNum;
+		}
+
+		while(nextNum < maxFibonacciNum){
+			if(nextNum % 2 == 0){
+				eventFibonacciNumSum += nextNum;
+			}
+			
+			firstNum = secondNum;
+			secondNum = nextNum;
+			nextNum = firstNum + secondNum;
+		}
+		
+		return eventFibonacciNumSum;
+	}
```

Comment:
I wrote the same logic in a simple way. Do not complicate things.
![image](https://cloud.githubusercontent.com/assets/3335673/9256871/ee2f901c-4211-11e5-8be9-03aecf41688c.png)